docs/source/advanced_usage.rst

```diff
@@ -281,37 +281,6 @@ we should always measure ``1``.
     Without Noise: [[1], [1], [1], [1], [1], [1], [1], [1], [1], [1]]
     With Noise   : [[0], [0], [0], [0], [0], [1], [1], [1], [1], [0]]
 
-Parametric Programs
-~~~~~~~~~~~~~~~~~~~
-
-A big advantage of working in pyQuil is that you are able to leverage all the functionality of
```

Comment:
Could we keep this section and state that it was removed, and what alternatives there are now?

Reply:
done